src/Analysim.Web/ClientApp/src/app/projects/project/project.component.html

```diff
@@ -9,7 +9,7 @@ <h3 class="project-title" *ngIf="(project.route | route:'projectname') as projec
                 </h3>
                 <ul class="project-authors-list">
                   <li class="project-author" *ngFor='let members of project.projectUsers | projectMember:"owner"' [routerLink]='["/profile/" + members.user.userName]'>
-                    <img class="project-author-thumbnail" src="../../../assets/img/default-profile.png" />
+                    <img class="project-author-thumbnail" [src]="members.user | profileImage"/>
```

Comment:
If this is the "View profile" page, it worked for me. However, it gave a filter error when the file extension was all caps (.JPG).

Reply:
Yes, that line and also line 14 of modal-member-list.component.html

Interesting, it just shows the default profile image still for me. I checked the profile images I uploaded and both of them have lowercase jpg extensions. I tried multiple browsers, clearing cache, and dotnet publish.

Reply:
> Yes, that line and also line 14 of modal-member-list.component.html
> 
> Interesting, it just shows the default profile image still for me. I checked the profile images I uploaded and both of them have lowercase jpg extensions. I tried multiple browsers, clearing cache, and dotnet publish.

Oh, I only saw it update on the user profile page but not on the project list. So that one probably needs something else.